src/Packetery/Module/Order/Controller.php

```diff
@@ -192,16 +194,23 @@ public function saveModal( WP_REST_Request $request ) {
 		}
 
 		$values = $form->getValues( 'array' );
+		if (
```

Comment:
Přišlo by mi to přehlednější jako s jednou if-else podmínkou na is_numeric( $values['packetery_weight'] )

Reply:
To bych tam pak měl extra vnořenou podmínku. Takže celkem tři bloky, Vnořené podmínky nechceme. Stejné váhy se mají ignorovat.

Reply:
Jde mi hlavně o to, že podmínka by se nevyhodnocovala dvakrát. A jako bonus by to mělo méně řádek.

Reply:
Dvě podmínky řeší dvě situace. Vnořené podmínky se blbě čtou. Nechci vnímat tří bloky kódu, když mohu číst jen dvě. Ta podmínka není náročná na výpočet.